### ISSUE TITLE
wrong generation of extensionpermissionset.xml

### ISSUE THREAD

**Issue by zacktkd** [microsoft/AL#5484]
I have an app for business central called 'Xyz'
I generate the file extensionpermissionset.xml
the values of the RoleID and RoleName attributes of the xml "PermissionSet" node are 'XYZ' and 'Xyz' respectively.
I change the name of the app to 'Abc'
I delete the file extensionpermissionset.xml
I generate the file extensionpermissionset.xml again
the values of the RoleID and RoleName attributes of the xml "PermissionSet" node are still 'XYZ' and 'Xyz' respectively (I would expect ABC and Abc)
I close the visual studio code, I open it again,
I generate the file extensionpermissionset.xml again and the RoleID and RoleName attribute values of the xml node "PermissionSet" are 'ABC' and 'Abc' respectively.

basically to find the right values you need to:
delete the file extensionpermissionset.xml
change the name of the app,
close and reopen visual studio code,
rebuild the file extensionpermissionset.xml.

i use : ﻿
# Business Central 2019 release wave 2 update (CURRENT - 2019/10)
Version 4.0.0
